pyproject.toml

```diff
@@ -228,3 +228,8 @@ required-imports = ["from __future__ import annotations"]
 
 [tool.ruff.flake8-import-conventions]
 extend-aliases = {xarray = "xr"}
+
+[tool.nbqa.addopts]
+ruff = [
+    "--extend-ignore=B018"
+]
```

Comment:
I don't know how to add `per-file-ignore` for `nbqa` (see https://github.com/nbQA-dev/nbQA/issues/730) but I think this does the same since we only have notebooks in the tutorials @marscher